Sources/Internal/FrameProvider.swift

```diff
@@ -47,8 +47,9 @@ final class FrameProvider {
     let insetWidth = monthWidth - content.monthDayInsets.left - content.monthDayInsets.right
     let numberOfDaysPerWeek = CGFloat(7)
     let availableWidth = insetWidth - (content.horizontalDayMargin * (numberOfDaysPerWeek - 1))
-    let points = availableWidth / numberOfDaysPerWeek
-    daySize = CGSize(width: points, height: points)
+    let width = availableWidth / numberOfDaysPerWeek
+    let height = width * content.dayAspectRatio
+    daySize = CGSize(width: width, height: height)
```

Comment:
always a pleasant surprise when you realize you didn't bake in any assumptions about the days being squares  - like a Christmas present from my past self 😂